python/hail/expr/aggregators.py

```diff
@@ -824,8 +824,8 @@ def call_stats(expr, variant):
 
     .. doctest::
 
-        >>> dataset_result = dataset.annotate_rows(gt_stats = agg.call_stats(dataset.GT, dataset.v))
-        >>> dataset_result.rows_table().select('v', 'gt_stats').show()
+        >>> dataset_result = dataset.annotate_rows(gt_stats = agg.call_stats(dataset.GT, dataset.alleles))
+        >>> dataset_result.rows_table().select('locus' 'alleles', 'gt_stats').show()
         +-----------------+--------------+----------------+-------------+--------------+
         | v               | gt_stats.AC  | gt_stats.AF    | gt_stats.AN | gt_stats.GC  |
```

Comment:
Need to regenerate?  v should probably be locus + alleles.

Reply:
yeah, I know. I haven't really done docs yet, am starting that now.